test/zebras/test_schema.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+import pytest
+
+import numpy as np
+
+import zebras as zb
+
+
+schema = zb.Schema(
+    {
+        "target": zb.Field(ndim=1, tdim=0, past_only=True),
+        "time_feat": zb.Field(
+            ndim=2, tdim=-1, past_only=True, preprocess=np.atleast_2d
+        ),
+        "static": zb.Field(ndim=1, preprocess=np.atleast_1d),
+    }
+)
+
+
```

Comment:
Tests for `load_splitframe`?

Reply:
Coming! Note, that this PR is still in draft mode :)